### PR TITLE
[TECHNICAL SUPPORT] LPS-181292 Description field overrides localisation

### PR DESCRIPTION
**Pull request by NemethNorbert**
Hey,

issue: [LPS-181292](https://issues.liferay.com/browse/LPS-181292),
caused by: [LPS-144199](https://issues.liferay.com/browse/LPS-144199)

The original issue described in [LPS-144199](https://issues.liferay.com/browse/LPS-144199) did not require this commit that I am about to revert. The root cause of this issue is a race condition, where the default change event from ckeditor is handled after we already changed the selectedLanguageId of the inputLocalized field. This results setting the previous locale's data to the newly selected one, thus corrupting existing locales or even new ones.
https://github.com/liferay/liferay-portal/blob/master/portal-web/docroot/html/taglib/ui/input_localized/page.jsp#L48-L54

Please review my changes.

Thanks,

### CODE REVIEW

**Comment by dsanz** [comment]
@markocikos please take a look

**Comment by markocikos** [comment]
Oh boy, this PR is extremely scary. In general, setting of data in CKEditor is one of the most fragile parts of DXP, with something like five layers of code modifying data and activating listeners. And now, we are adding and arbitrary timeout to fix a race condition :fearful:  Are there any alternative approaches or workarounds to this solution?

**Comment by NemethNorbert** [comment]
Hey @markocikos ,

It is indeed scary. That's the reason I decided to revert the change from [LPS-144199](https://issues.liferay.com/browse/LPS-144199). It seemed the safest solution, considering it was working trough 7.0-7.4-u35

**Comment by markocikos** [comment]
Hey @NemethNorbert ! I'd like to dive more deeply into this, see if we can hack some alternative solution of workaround. But, I am swamped with other tasks atm. 

We could push this as a maybe-temporary solution. I am not very comfortable with it, it may cause regressions or complications in the future. But CI is green, so I may be wrong. @dsanz what do you think?

**Comment by dsanz** [comment]
mmm this is tricky. Looks like the react wrapper allows us to get rid of the `checkdirty()` , but the JSP way (`ckeditor.jsp`)  requires us to have 2 hacky things:
 * use of `checkDirty()`, apparently not needed as per [this comment](https://github.com/liferay-frontend/liferay-portal/pull/2549#issue-1309703172)
 * `setInterval()` which continously calls the onchange(), hence the need of `checkDirty()`

In light of this, it seems like the traditional way of creating the ckeditor, which we exercise in the JSP, is not offering the convenience semantics we need. This was an issue in the past, where [similar solutions](https://alfonsoml.blogspot.com/2011/03/onchange-event-for-ckeditor.html) were adopted out there. The more recent `change` event looks [not very reliable either](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#event-change) and does not work in source mode.

If we had this in the past, and we can't find a more reliable way, I'm ok with keeping it after the corresponding tests have passed. We will need to revisit all this at some point, but probably with a full migration to a newer version.

So, in short, I'd keep separate strategies for `classic_ckeditor` and `ckeditor` and ensure this gets tested.

**Comment by markocikos** [comment]
ci:forward
